libsplinter/src/orchestrator/mod.rs

```diff
@@ -29,7 +29,7 @@ use protobuf::Message;
 
 use crate::channel;
 use crate::error::InternalError;
-use crate::mesh::{Envelope, Mesh, RecvTimeoutError as MeshRecvTimeoutError};
+use crate::mesh::{Envelope, Mesh, RecvTimeoutError as MeshRecvTimeoutError, SendError};
```

Comment:
This fix seems unrelated, could go in another separate PR?

Reply:
moved to https://github.com/Cargill/splinter/pull/1309